doc/changelog/10-standard-library/13582-exp_ineq.rst

```diff
@@ -0,0 +1,9 @@
+- **Changed:**
+  Minor Changes to Rpower:
+  Generalizes exp_ineq1 to hold for all non-zero numbers.
+  Adds exp_ineq1_le, which holds for all reals (but is a <= instead of a <).
+
+  (`#13582 <https://github.com/coq/coq/pull/13582>`_,
+  by Avi Shinnar and Barry Trager, with help from Laurent Théry
+
```

Comment:
The white line an final parenthesis seems to generate errors when adding new changelogs after this one #13671 .
If I am right and since this is already merged, what is the best way to make a modification? create an adhoc new PR?

Reply:
Yes, you are right about the diagnosis. You can either open an ad hoc PR or fix it in your own PR, especially if your own PR shouldn't take too long to merge.